src/lib.rs

```diff
@@ -93,6 +94,7 @@ fn _pydantic_core(py: Python, m: &PyModule) -> PyResult<()> {
     m.add_class::<PyMultiHostUrl>()?;
     m.add_class::<ArgsKwargs>()?;
     m.add_class::<SchemaSerializer>()?;
+    m.add_class::<TzInfo>()?;
```

Comment:
does this actually need to be exported? If it does, we should have a test for importing it.

Reply:
It does so that it can be unpickled

Reply:
The pickle test is already testing that it can be imported